hermes-eventbus/src/main/java/xiaofei/library/hermeseventbus/HermesEventBus.java

```diff
@@ -287,22 +298,35 @@ public void call(IMainService o) {
                 }
             });
              */
-            IMainService mainService = Hermes.getInstanceInService(service, IMainService.class);
-            mainService.register(Process.myPid(), SubService.getInstance());
-            mRemoteApis.set(mainService);
-            mState = STATE_CONNECTED;
+
+//            Log.v(TAG, "Hermes " + Service.class.getCanonicalName());
+//            Log.v(TAG, "Hermes " + service.getCanonicalName());
+            if(service.getCanonicalName().equals(Service.class.getCanonicalName())) {
```

Comment:
The purpose of this `if` is that the following code
```
IMainService mainService = Hermes.getInstanceInService(service, IMainService.class);
mainService.register(Process.myPid(), SubService.getInstance());
mRemoteApis.set(mainService);
mState = STATE_CONNECTED;
```
should only get triggered if `public void onHermesConnected` is for HermesEventBus Service only, otherwise 
```
IMainService mainService = Hermes.getInstanceInService(service, IMainService.class);
mainService.register(Process.myPid(), SubService.getInstance());
mRemoteApis.set(mainService);
mState = STATE_CONNECTED;
```
will always get trigger for any Hermes connect call.